examples/k8s.yaml

```diff
@@ -123,7 +123,7 @@ provision:
     kubectl apply -f https://github.com/flannel-io/flannel/releases/download/v0.22.1/kube-flannel.yml
     # Control plane node isolation
     kubectl taint nodes --all node-role.kubernetes.io/control-plane-
-    sed -e "s/${LIMA_CIDATA_SLIRP_IP_ADDRESS:-192.168.5.15}/127.0.0.1/" -i $KUBECONFIG
+    sed -e "/server:/ s|https://.*:\([0-9]*\)$|https://127.0.0.1:\1|" -i $KUBECONFIG
```

Comment:
nit: maybe we need a comment line to explain this regexp

Reply:
Added a line to explain what we are replacing here.

Once we implement multi-node, we might need to be more careful so that the other nodes can still talk to the apiserver even if we edit the config to work with the tunneled port from the host side. Same goes for k3s, then.